var/spack/repos/builtin/packages/globalarrays/package.py

```diff
@@ -25,42 +25,59 @@
 from spack import *
 
 
-class Globalarrays(CMakePackage):
-    """The Global Arrays (GA) toolkit provides a shared memory style
-    programming environment in the context of distributed array data
-    structures.
-    """
+class Globalarrays(AutotoolsPackage):
```

Comment:
can't you use `cmake` build? Or does it still need heavy patching?

Reply:
The patches were for the IBM XL compiler to work, but the lines patched no longer match up with the current implementation. We have an ongoing effort to get the cmake build running again, but for now our support build tool is autotools.

Reply:
fair enough